go/libraries/doltcore/table/untyped/sqlexport/sqlwriter.go

```diff
@@ -145,6 +160,98 @@ func (w *SqlExportWriter) insertStatementForRow(r row.Row) (string, error) {
 	return b.String(), nil
 }
 
+func (w *SqlExportWriter) WriteDeleteRow(ctx context.Context, r row.Row) error {
+	var b strings.Builder
+	b.WriteString("DELETE FROM ")
+	b.WriteString(sql.QuoteIdentifier(w.tableName))
+	b.WriteString(" ")
+
+	b.WriteString(" WHERE ( ")
+	seenOne := false
+	_, err := r.IterSchema(w.sch, func(tag uint64, val types.Value) (stop bool, err error) {
+		col := w.sch.GetAllCols().TagToCol[tag]
+		if col.IsPartOfPK {
+			if seenOne {
+				b.WriteString(" AND ")
+			}
+			sqlString, err := w.sqlString(val)
+			if err != nil {
+				return true, err
+			}
+			b.WriteString(col.Name)
```

Comment:
Some column names will not be valid sql identifiers (will include spaces, etc.). You'll want to escape all column names, here and elsewhere with backticks (`colName`).

Also add a test to ensure this works

Reply:
Looks like I already wrote sql.QuoteIdentifier for this purpose

Reply:
done